tests/unit_tests/data_steward/utils/sandbox_test.py

```diff
@@ -19,3 +19,7 @@ def test_get_sandbox_table_name(self):
         expected = f'{self.table_namer}_{base_name}'
         actual = sandbox.get_sandbox_table_name(self.table_namer, base_name)
         self.assertEqual(expected, actual)
+        self.assertEqual(base_name,
+                         sandbox.get_sandbox_table_name(None, base_name))
+        self.assertEqual(base_name,
+                         sandbox.get_sandbox_table_name('', base_name))
```

Comment:
Nice unit tests.  Consider adding one to test how you want it to behave if someone passes in a spaces only string.

Reply:
Will add this.